frontend/src/resources/utils/get-cluster.test.ts

```diff
@@ -156,7 +161,7 @@ const mockManagedClusterInfoFailedUpdating: ManagedClusterInfo = {
                 version: '1.2.3',
                 availableUpdates: [], //deprecated
                 versionAvailableUpdates: [{ version: '1.2.4', image: 'abc' }],
-                desiredVersion: '', //deprecated
+                desiredVersion: '1.2.3', //deprecated
```

Comment:
I'm using the MCI's "desired version" field in the get-cluster.ts logic and it seems to be in use within our daily cluster. @KevinFCormier Any idea if this deprecated note is accurate? And if it is, does it mean we should change our approach here?

Reply:
Yes, see explanation in other comment. If you want to test the fallback path, you'll need one MCE with `desiredVersion` and without `desired.version` and vice versa.